### ISSUE TITLE
Identify some solid concept art for further consideration

### ISSUE THREAD

**Issue by cubap** [cubap/trade#48]

**Comment by ladmeyer** [comment]
An interesting art style would probably not be one exactly like any other games like this one, with exploration, survival, i.e. An art style I think that hasn't been done yet (and that should be done) is a pixelated version of real objects. I know of an "engine" that can do this with photos. It isn't perfect, but I think it may be what Trade needs for an art style.

Locke

**Comment by cubap** [comment]
I kind of like the idea of bitmaps of real photos. The textures would be repeated, though, so it may take some doing to make it work well. Too pixelated would start looking voxel-y like Minecraft and clones, but that is a direction we could look at.

My brother's an artist. I'll talk to him and see if he's willing to donate some time to me.

**Comment by ladmeyer** [comment]
Right. I'm thinking we pixelate them an then edit them, to make them look more video game-esque. It might also be cool to have some computer-generated randomness to the textures, because all trees don't look the same in real life.

**Comment by cubap** [comment]
Punted. Concept art is not in scope for this project.
